Moving math equations from screenshots into MathJax

poliwop opened the pull request:
@inventandchill I have put the equations into the notebook itself, please review that I have transcribed them correctly. Once you confirm that I have, I will remove the images from the file, and then complete this merge.

inventandchill commented:
@poliwop  created new file with all changes and removed pictures
https://github.com/galacticcouncil/HydraDX-simulations/blob/cg-optimal-arbitrage-math/hydradx_update/derivations/Optimal_arbitrage_v2.ipynb

poliwop commented:
@inventandchill I moved your changes to the original doc. I also removed the fee version since I believe it was wrong. If you want to work on a corrected version with fee, you can do that in this branch before we merge it up maybe?

poliwop commented:
I will try to help out too if I have time, with the fee version
